manifests/init.pp

```diff
@@ -105,6 +105,14 @@
 # $autosign_mode::                          mode of the autosign file/script
 #                                           type:Pattern[/^[0-9]{4}$/]
 #
+#
+# $autosign_content::                       If set, write the autosign file content
```

Comment:
As you can see in the latest commit, in server.pp I also added 'type:string'. I hope this is OK.